### DISCUSSION TITLE
Configuring custom nginx page

### DISCUSSION THREAD

**Discussion by BiggVecc** [searx/searx#2432] [Installation issue: Official wiki] [answered]
Hello, 

I don't get the procedure to setup a custom nginx page with filtron+monty.

I ran the various scripts in the wiki and I've created an nginx site in the sites-available directory. 
I edited the file with my domain name and I added the filtron + monty parts inside the server configuration to run the instance with both the proxies. 
As soon as I restarted the nginx service I noticed that the page stay in the default state (welcome to nginx!).
The searx page only pops up on http://mydomain/searx, which at least works, but since I've created a subdomain with https I would like to run the instance on that subdomain.

If this answer is present on the wiki feel free to send me to rtfm and a link to the section :)

I'll post below my nginx site config:
server {

server_name searx.domain.com;

location /searx {
proxy_pass http://127.0.0.1:4004/;

proxy_set_header   Host             $http_host;
proxy_set_header   Connection       $http_connection;
proxy_set_header   X-Real-IP        $remote_addr;
proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
proxy_set_header   X-Scheme         $scheme;
proxy_set_header   X-Script-Name    /searx;

}

location /searx/static {
alias /usr/local/searx/searx-src/searx/static;
}

location /morty {
proxy_pass http://127.0.0.1:3001/;

proxy_set_header   Host             $http_host;
proxy_set_header   Connection       $http_connection;
proxy_set_header   X-Real-IP        $remote_addr;
proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
proxy_set_header   X-Scheme         $scheme;

}

    listen 443 ssl; # managed by Certbot
    ssl_certificate /etc/letsencrypt/live/searx.domain.com/fullchain.pem; # managed by Certbot
    ssl_certificate_key /etc/letsencrypt/live/searx.domain.com/privkey.pem; # managed by Certbot
    include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot

}
server {
    if ($host = searx.domain.com) {
        return 301 https://$host$request_uri;
    } # managed by Certbot

server_name searx.domain.com;
    listen 80;
    return 404; # managed by Certbot

}

Thank you very much, 

BV

**Comment by JacksonTaylorxyz** [comment, marked as answer]
I also have been largely unsuccessful in setting up searx on my own subdomain. But with your config, have you tried searx.domain.com/searx?

**Reply by BiggVecc** [reply]
Oof.... You found the detail that I totally underestimated. Thank you very much @JacksonTaylorxyz now it works.
Now I'm feeling like an idiot since I wasted an entire night of sleep just to find this small detail.

Thaks Again!
